aip/general/0202.md

```diff
@@ -81,6 +81,39 @@ backwards compatible.
 Any new `FieldInfo.Format` value **must** be governed by an
 [IETF-approved RFC][ietf rfc] or a [Google-approved AIP](./0001.md).
 
+### Type References
+
+Some generic field types (AIP-146), like `google.protobuf.Any`, can still have
+a limited set of possible types. Identifying the type(s) of said field can be
+done with the `google.api.TypeReference` via the
+`google.api.FieldInfo.referenced_types` field, like so:
+
+```proto
+message ModelResponse {
+  google.protobuf.Any raw_response = 1 [(google.api.field_info) = {
+    referenced_types: {type_name: "google.library.v1.TitleGenerationResponse"}
+    referenced_types: {type_name: "google.library.v1.ChapterGenerationResponse"}
+    referenced_types: {type_name: "google.library.v1.SummaryGenerationResponse"}
+  }];
+}
+```
+
+If the set of possible types is unknown to the specification author i.e., the
```

Comment:
Nit: move the comma to before the `i.e.` (and possibly turn it into `e.g.` - I don't think this is the *only* situation where we'd want `*`; Cloud Audit Logs springs to mind - and indeed LROs). CloudEvents use Any as well...